darts/models/forecasting/ensemble_model.py

```diff
@@ -362,6 +383,22 @@ def find_max_lag_or_none(lag_id, aggregator) -> Optional[int]:
     def _models_are_probabilistic(self) -> bool:
         return all([model._is_probabilistic() for model in self.models])
 
+    def _models_same_likelihood(self) -> bool:
```

Comment:
for quantileregression we also need to check that the same quantiles were fitted, right?

Reply:
correct